### PR TITLE
Add AB Docs link to readme, + other nits

### PR DESCRIPTION
**Pull request by ryley-o**
Update this repo's readme with a link to AB Docs site, which now helps instruct users how to perform subgraph queries.

This change also updates a nit detail in the `Adding New Minters Checklist` section of our readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 The Art Blocks subgraph definitions for The Graph.
 
+For more information on Art Blocks subgraph schema and queries, see the [Art Blocks documentation](https://docs.artblocks.io/creator-docs/art-blocks-api/queries/) website.
```
Comment (lindsgil) [inline README.md:5]: 🙌

**Review by lindsgil — APPROVED** [review]

**Comment by ryley-o** [comment]
@lindsgil - ty for the quick review! I added one more file in 58d816b to hopefully help us remember to keep docs in sync with this too!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Description of the change
+
+\<add description here\>
+
+>reminder: Any subgraph deployments should be documented as [Releases](https://github.com/ArtBlocks/artblocks-subgraph/releases) in this repository.
+
+>reminder: Any changes to subgraph schema should be accompanied by corresponding changes to the Art Blocks documentation site, available at https://docs.artblocks.io/ (specifically, see the [Subgraph Entities](https://docs.artblocks.io/creator-docs/art-blocks-api/entities/) and [Subgraph Querying and API Overview](https://docs.artblocks.io/creator-docs/art-blocks-api/queries/) sections).
```
Comment (lindsgil) [inline .github/pull_request_template.md:7]: 🥳

**Comment by lindsgil** [comment]
> @lindsgil - ty for the quick review! I added one more file in [58d816b](https://github.com/ArtBlocks/artblocks-subgraph/commit/58d816b900919e8381488e8c2d0a20edfec40ecd) to hopefully help us remember to keep docs in sync with this too!

awesome! great call :) thanks for the update!

**Review by jakerockland — APPROVED** [review]

**Review by lyaunzbe — APPROVED** [review]

**Pull request merged**